### ISSUE TITLE
Encryption Export question during App Submittal to App Store.

### ISSUE THREAD

**Issue by nin9creative** [Simbul/baker#910]
I'm working with someone that is going through the submittal process and I had a basic question for those who've already submitted Baker apps.

During the app submittal process, there is the question that pops up about Encryption technology used in the application.  I've never submitted an app that used encryption so I've never dealt with this. 

I know that Baker uses a zip library that includes some encryption code, but from the comments in that file looks like it is exempt and ok for export.  So if you were submitting a basic Baker app you should be good to go to answer "No".

However let's just say in the case of someone developing a server connection to a backend (such as MagRocket)...  If a developer decided to install SSL certs and connect to the backend Baker API via HTTPS, then I think this would force the developer to answer "Yes" to the export question and follow the necessary process to get certification/approval etc... As the use of HTTPS for a transport layer would classify the application as using encryption technology. If the developer connected via simple HTTP then again the answer would be "No".

Info related to this...
http://tigelane.blogspot.com/2011/01/apple-itunes-export-restrictions-on.html

Just thought I'd put this out there for discussion....

**Comment by folletto** [comment]
You're correct about the zip library, but even if that would require authorization, Baker doesn't use the encryption part and as such doesn't get complied, so it's fine.

But if you use encryption in any way (including, yes, HTTPS)... you get into one of the blurry grounds there. Blurry in the sense that yes, if you use encryption you should declare it, but it's unlikely that something will happen if you don't report.

So: with HTTPS you should answer YES.
